styles/main.scss

```diff
@@ -41,6 +41,12 @@ $fontFamily: "system-ui";
 
 @import "./dividers.scss";
 
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
```

Comment:
It looks like I still need this line : ( The vertical scrollbar only shows up on a big monitor.